rbc/tests/test_omnisci_array_methods.py

```diff
@@ -19,13 +22,27 @@ def omnisci():
 ndarray_methods = [
     ('fill', (5, 4), [4.0, 4.0, 4.0, 4.0, 4.0]),
     ('max', (5, 4.0), 4.0),
-    ('max_empty', (0, ), -127),
+    ('max_empty_int8', (0, ), np.iinfo(np.int8).min + 1),
+    ('max_empty_int16', (0, ), np.iinfo(np.int16).min + 1),
+    ('max_empty_int32', (0, ), np.iinfo(np.int32).min + 1),
+    ('max_empty_int64', (0, ), np.iinfo(np.int64).min + 1),
+    ('max_empty_float32', (0, ), np.finfo(np.float32).min),
+    ('max_empty_float64', (0, ), np.finfo(np.float64).min),
     ('max_initial', (5, 4.0, 30.0), 30.0),
     ('mean', (5, 2.0), 2.0),
-    ('mean_empty_float', (0, ), np.nan),
-    ('mean_empty_int', (0, ), 0),
+    ('mean_empty_int8', (0, ), 0),
```

Comment:
Hmm, according to `omnisci_array_mean`, the result ought to be `np.nan`. Is `mean_empty_int8` actually tested?

Reply:
Yes! Because the return type is an integer type, llvm cast np.nan to an integer value, which happens to be 0

Reply:
ok, so when `mean` will return a float, we'll have `nan` here.

Reply:
exactly